### PR TITLE
Added content for embedded devices.

### PR DESCRIPTION
**Pull request by frenzymadness**
SSIA

### CODE REVIEW

File: points.yml [hunk 1]
```diff
@@ -29,10 +29,12 @@
 - name: Embedded Stuff
   ready: false
   content: |
-    Push Python to all the devices, all from Fedora!
+    Programming for computers and servers can be fun, but what about controlling much smaller devices such as Micro bit, boards with the ESP8266 chip or PyBoard? With Fedora, you have everything you need, and the world of the Internet of things is opened for you.
```
Comment (hroncok) [inline points.yml]: BBC micro:bit, pyboard

File: points.yml [hunk 2]
```diff
@@ -29,10 +29,12 @@
...
+
+    Do you just want to play with MicroPython and spot the differences? No problem, just install MicroPython from the repository.
```
Comment (hroncok) [inline points.yml]: This info was requested by me, but it kills the message. Maybe something like:

> Even without the actual device, you can run MicroPython on Fedora to test you code.

**Comment by hroncok** [comment]
I've sent a pull request for you at https://github.com/frenzymadness/fedoralovespython.org/pull/1

**Comment by frenzymadness** [comment]
Your PR is closed, changes applied.

**Pull request closed without merging**